installer_scripts/install-yb-voyager

```diff
@@ -953,7 +953,7 @@ ubuntu_install_postgres() {
 	sudo apt install -y postgresql-common 1>&2
     echo | sudo /usr/share/postgresql-common/pgdg/apt.postgresql.org.sh 1>&2
 	sudo apt-get update 1>&2
-	sudo apt-get -y install postgresql-17  1>&2
+	sudo apt-get -y install postgresql-client-17 1>&2
```

Comment:
So now we won't be installing postgresql server on the client machine which is better.

Q: What if there is a existing pg_dump of the machine, does this override or skip?
Q: Here we are mentioning `17`, will it install the latest release in that series?
Q: If yes, shall/can we fix it to some `17.x` version?

Reply:
1) On **rhel9**, I installed **16** and then install **17**: it was able to update to the latest **17**. Then I tried installing **17** first and then **16** and it did not downgrade the binary to **16**.
On **ubuntu**, I havent been able to text explicitly yet. But the installer script does update the binary from **17.2** to **17.3** on the jenkins machine
2) **Yes**, it does install the **latest** in that series
3) On **ubuntu**, it seems like only latest package in the series i available to install. I was not able to install **17.2** explicitly or even find that package.
On **rhel9**, I was able to do it though. But if we are not able to do it on ubuntu then we should probably skip it on rhel too.